homeassistant/components/starlink/binary_sensor.py

```diff
@@ -64,62 +64,70 @@ def is_on(self) -> bool | None:
         device_class=BinarySensorDeviceClass.UPDATE,
         value_fn=lambda data: data.alert["alert_install_pending"],
     ),
-    StarlinkBinarySensorEntityDescription(
-        key="roaming",
-        name="Roaming mode",
-        value_fn=lambda data: data.alert["alert_roaming"],
-    ),
     StarlinkBinarySensorEntityDescription(
         key="currently_obstructed",
         name="Obstructed",
         device_class=BinarySensorDeviceClass.PROBLEM,
         value_fn=lambda data: data.status["currently_obstructed"],
     ),
+    StarlinkBinarySensorEntityDescription(
+        key="roaming",
+        name="Roaming mode",
+        entity_registry_enabled_default=False,
+        value_fn=lambda data: data.alert["alert_roaming"],
+    ),
     StarlinkBinarySensorEntityDescription(
         key="heating",
         name="Heating",
         entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
```

Comment:
So, the entities we generally aim to disable by default, are the ones that are less commonly used AND update high frequently. For example, voltages.

In all other cases, the entity category will handle it already, so they don't need to be disabled by default.

This sensor is a good example of something I don't expect to update at a high frequency and has a diagnostic entity category assigned. For that reason, there is no reason to disable it by default?

Considering the above, you might want to review the others too.

Reply:
Gotcha, I have kept the diagnostic binary sensors as enabled, since practically none of them should be updating often. There's a couple of diagnostic sensors that are disabled, those are for the dish azimuth & elevation, which changes pretty much constantly. Thanks!